bin/test-workflow/stop

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+source utils.sh
+
+if [[ "${CONJUR_OSS_HELM_INSTALLED}" == "true" ]]; then
+  kubectl delete namespace $CONJUR_NAMESPACE_NAME
```

Comment:
`CONJUR_NAMESPACE_NAME` on this line and `$TEST_APP_NAMESPACE_NAME` on next line should be double quoted.